.gitignore

```diff
@@ -31,6 +31,16 @@ test/*.cmp
 t_*
 bin
 vw
+!java/
```

Comment:
These are all necessary as the JNI classes start with VW and that is excluded by the above line.  If anyone knows of a prettier and more robust way to do this that would be great.